source/wp-content/themes/wporg-developer-2023/src/code-title/style.scss

```
@@ -1 +1,39 @@
 /* css styles */
+.wp-block-wporg-code-reference-title {
+	> * {
+		font-size: var(--wp--preset--font-size--extra-large);
+		line-height: 42px;
```

Best practice to use unitless, see https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#values
```suggestion
		line-height: 1.73;
```